### PR TITLE
#162727477 list users functionality

### PR DESCRIPTION
**Pull request by daniellamarr**
#### What does this PR do?
- This PR list all available artists on the app
- This PR shows the profile of a particular artist
#### Description of Task to be completed?
- Authenticated users should be able to view a list of artists on the app
- Authenticated users should be able to view the profile of a particular artist on the app
#### How should this be manually tested?
- Run `npm i`
- Run `npm run db:migrate`
- Send a **GET** request to `/api/v1/users/artists`, you must be logged in and a token must be passed to the request before you can access the route. The route returns a list of all artists on the platform.
- Send a **GET** request to `/api/v1/users/artists/:artistId`, where `artistId` is the artist Id. Pass in access token before accessing the route. The route returns profile of a particular artist based on ID passed.
#### Any background context you want to provide?
- User must have signed up and logged in on the app before attempting to test this feature.
#### What are the relevant pivotal tracker stories?
[#162727477](https://www.pivotaltracker.com/story/show/162727477)
#### Screenshots (if appropriate)
screen shot 2019-01-17 at 12 46 17 am
screen shot 2019-01-17 at 12 57 29 am

#### Questions:
N/A

### CODE REVIEW

**Review by Proception — COMMENTED** [review]
Looks good to me, however, resolve conflicts

**Review by julietezekwe — APPROVED** [review]
Looks good

**Review by wombolo — APPROVED** [review]
LGTM

**Review by abejide001 — APPROVED** [review]
LGTM

File: src/controllers/UsersController.js [hunk 1]
```diff
@@ -252,6 +252,104 @@ class UsersController {
...
+   * @param {object} res
+   * @memberof UsersController
+   * @returns one artist on the platform
+   */
+  static async getOneArtist(req, res) {
+    try {
+      const { artistId } = req.params;
+
+      /* eslint-disable no-restricted-globals */
+      if (isNaN(artistId)) {
```
Comment (andela-dbamidele) [inline src/controllers/UsersController.js]: Can you move this to a middleware?

**Review by andela-dbamidele — CHANGES_REQUESTED** [review]

File: src/controllers/UsersController.js (hunk 1 above)
Comment (daniellamarr) [reply src/controllers/UsersController.js]: Alright

Comment (daniellamarr) [reply src/controllers/UsersController.js]: Done

**Review by andela-dbamidele — APPROVED** [review]
And also, please ensure that your commit messages describe the exact changes that you made.

**Pull request merged**